### DISCUSSION TITLE
Collage template for portrait setup

### DISCUSSION THREAD

**Discussion by Florian-Costel** [photobooth-app/photobooth-app#318] [Howtos and camera specific topics]
Hi,
Can anyone point me in the right direction to create a collage as in attached image? I need to create a collage with two images and printed in portrait format. Is there a file where I can setup the image format for collage?

Thanks,
Florian

![template-portrait](https://github.com/photobooth-app/photobooth-app/assets/175205152/7439bb1a-83c8-44d7-a7a8-27a868d46b8c)

**Comment by mgineer85** [comment]
That should be easy:

change the existing collage settings
Adjust collage height/width setting as needed to make it portrait.
The images used are defined in the "merge definition". There you can set the X/Y position and so on. Just fiddle around with the values until it's satisfying.
![image](https://github.com/photobooth-app/photobooth-app/assets/3868450/900a1352-de8d-4fcf-921e-34f12b378e71)

**Comment by Florian-Costel** [comment]
Thank you very much! Works like a charm!
